runtime/onert/core/src/exec/Executors.cc

```diff
@@ -87,21 +87,58 @@ void Executors::execute(const IODescription &desc)
 
 void Executors::checkSupportedMultimodel() const
 {
-  // Assume 2 executors only
-  // Assume that each model may have only one subgraph
-  // TODO Support general case
-  if (_executors.size() != 2)
-    throw std::runtime_error{"NYI: Multi model execution for this package is not supported yet"};
+  // Assumption
+  // Models
+  //   size: n
+  //   sizeof(model(a).outputs) == sizeof(model(a+1).inputs)
+  auto const model_count = modelCount();
 
-  // Assume all edges are 0:0:x -> 1:0:x
+  for (auto &pair : _executors)
```

Comment:
@hseok-oh If there is no executor for model_index _a_ in [0, model_count), this loop seems not detect error. If you expects all models are connected sequentially, what about just iterate with integer index?